projects/vdk-core/cicd/build.sh

```diff
@@ -41,8 +36,8 @@ pip install --use-deprecated="legacy-resolver" -e .
 # Below line uses --use-deprecated=legacy-resolver to temporary workaround a
 # dependency backtracking issue.
 # TODO: Remove the use of the legacy resolver once latest pip works as expected.
-echo "Install common vdk test utils library (in editable mode)"
-pip install --use-deprecated="legacy-resolver" -e ../vdk-plugins/vdk-test-utils
+echo "install dependencies from requirements.txt (used for development and testing)"
+pip install --use-deprecated="legacy-resolver" --extra-index-url $PIP_EXTRA_INDEX_URL -r requirements.txt
```

Comment:
It's not necesssary to install vdk-test-utils as it's in requirements.txt

Change the order since installing the requirements.txt dependencies install vdk-core from pypi.org as well. But we want the one installed from source.

Reply:
Actually, the change includes removing the `pip install -e  ../vdk-plugins/vdk-test-utils`, that would rather install the potentially changed vdk-test-utils in the scope of a PR, instead of fetching the latest utils release. Why is that changed?

Reply:
Also, do we now drop --use-deprecated="legacy-resolver", or is todays pip version working correctly, or are you descoping this issue from the 3.11 upgrade?

Reply:
We can drop it separately after this change